SPECS/bazel/bazel.spec

```diff
@@ -46,7 +48,7 @@ cp ./scripts/packages/bazel.sh %{buildroot}/%{_bindir}/bazel
 %attr(0755,root,root) %{_bindir}/bazel-real
 
 %changelog
-* Tue Mar 05 2024 Riken Maharjan <rmaharjan@microsoft.com> - 6.5.0
+* Tue Mar 05 2024 Riken Maharjan <rmaharjan@microsoft.com> - 6.5.0-1
```

Comment:
Maybe now also mention the addition of the JDK patch.

Reply:
fixed